portal-web/test/functional/com/liferay/portalweb/macros/OpenIDConnect.macro

```diff
@@ -49,6 +54,38 @@ definition {
 		}
 	}
 
+	macro addProviderInInstanceSettings {
+		GlobalMenu.gotoPortlet(
+			category = "Configuration",
+			panel = "Control Panel",
+			portlet = "Instance Settings");
+
+		SystemSettings.gotoConfiguration(
+			configurationCategory = "SSO",
+			configurationName = "OpenID Connect Provider",
+			configurationScope = "Virtual Instance Scope");
```

Comment:
Since `addProviderInInstanceSettings` is mostly the same as `addProviderInSystemSettings`, we should remove pull the navigation parts of this macro, put them in the test case level, and have a generic `addProvider` macr